platform/mellanox/rules.mk

```diff
@@ -51,4 +51,9 @@ $(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(APPLIBS) $(SX_COMPLIB) $(SXD_LIBS) $(SX_
 # Force the target bootloader for mellanox platforms to grub regardless of arch
 TARGET_BOOTLOADER = grub
 
+# Set y to include non upstream patches for mellanox
+INCLUDE_MLNX_PATCHES = n
```

Comment:
Why is this needed? Can’t patches for other subsystems be included?

Reply:
Other subsystems planning to include no-upstream patches can use EXTERNAL_KERNEL_PATCHES  flag to provide a public accessible link to the tarball